docs/Music Macro Language.md

```diff
@@ -0,0 +1,989 @@
+| 02 | cdelay | sample    | ldchan | stcio | ldcio | subio | ldio | stio     | testlayer | freelayer  |    | ldseqtoptr  | dyntbl       | sustain     | vibdepthgrad | *rbltz        |
+| 03 | cdelay | sample    | ldchan | stcio | ldcio | subio | ldio | stio     | testlayer | freelayer  |    | ldfilter    | short        | bend        | vibdelay     | *rbeqz        |
+| 04 | cdelay | sample    | ldchan | stcio | ldcio | subio | ldio | stio     | testlayer | freelayer  |    | ptrtodyntbl | noshort      | reverb      | dyncall      | *rjump        |
+| 05 | cdelay | sample    | ldchan | stcio | ldcio | subio | ldio | stio     | testlayer | freelayer  |    | dyntbltoptr | dyntbllookup |             | rvrbidx      | *bgez         |
+| 06 | cdelay | sample    | ldchan | stcio | ldcio | subio | ldio | stio     | testlayer | freelayer  |    | dyntblv     | font         |             | samplbook    | *break        |
+| 07 | cdelay | sample    | ldchan | stcio | ldcio | subio | ldio | stio     | testlayer | freelayer  |    | randtoptr   | stseq        | vibfreq     | ldparams     | *loopend      |
+| 08 | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    | randvel     | sub          | vibdepth    | params       | *loop         |
+| 09 | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    | randgate    | and          | releaserate | notepri      | *bltz         |
+| 0A | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    |             | mutebhv      | env         | stop         | *beqz         |
+| 0B | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    | ptradd      | ldseq        | transpose   | fontinstr    | *jump         |
+| 0C | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    | randptr     | ldi          | panweight   | reset        | *call         |
+| 0D | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    | instr       | stopchan     | pan         | gain         | *delay        |
+| 0E | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    |             | lditoptr     | preqscale   | bendfine     | *delay1       |
+| 0F | cdelay | sampleptr | ldchan | stcio | ldcio | subio | ldio | rldlayer | ldlayer   | dynldlayer |    |             | stptrtoseq   | vol         |              | *end          |
+
+Instructions marked with an asterisk (\*) are documented in the section *Branching Instructions*.
+
+**cdelay** *ticks*
+
+**Description:** Delays processing of the channel script by the provided number of ticks (up to 16).  A delay of 0 is effectively a no-op.
```

Comment:
why `c` in `cdelay`? Does that mean `channel`? Would `chandelay` make more sense for consistency? Is that `c` the same in `stcio` and `ldcio`?

Reply:
TBD.